test/unit/test_label_procs.py

```diff
@@ -0,0 +1,186 @@
+from __future__ import annotations
+
+import pytest
+from common_utils import generate_unique_name
+from common_utils import run_proc
+from common_utils import validate_row_count
+
+
+def test_smoke_create_drop_label(conn_cnx, timestamp_string):
+    label = generate_unique_name("label", timestamp_string)
+    sql = f"call ADMIN.CREATE_LABEL('{label}', NULL, NULL, 'rows_produced > 100');"
+
+    # create_label returns NULL in case of successful label creation
+    assert (
+        run_proc(conn_cnx, sql)[0] is None
+    ), "Stored procedure did not return NULL value!"
```

Comment:
nit: the parens around the `assert` condition are generally unnecessary (here and below)

Reply:
I think to have them on separate lines they are neccessary

Reply:
That's python's linter (ruff) doings ... 

```
ruff.....................................................................Failed
- hook id: ruff
- exit code: 1
- files were modified by this hook

Found 1 error (1 fixed, 0 remaining).

black....................................................................Failed
- hook id: black
- files were modified by this hook
```


```
     sql = f"call ADMIN.DELETE_LABEL('{label}');"
-    assert run_proc(conn_cnx, sql)[0] == "done", "Stored procedure did not return NULL value!"
+    assert (
+        run_proc(conn_cnx, sql)[0] == "done"
+    ), "Stored procedure did not return NULL value!"
```